gp_emulator/gpu/gpu_predict.h

```diff
@@ -0,0 +1,94 @@
+/* A file to test imorting C modules for handling arrays to Python */
+
+#include "Python.h"
+#include "arrayobject.h"
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+/* Includes, cuda */
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <cublas_v2.h>
+#define DOUBLE__PRECISION
```

Comment:
Let CMAKE define quantities like this. Use a CMAKE variable so you can set these with ccmake. Then import the definition into your code with a cmake .in file.